spec/integration/categories_spec.rb

```diff
@@ -1,14 +1,6 @@
 require 'rails_helper'
 
```

Comment:
- [ ] OPTIONAL
There are a few spec test cases failing! Kindly ensure that **all** specs are passing!

**Failing specs:**
![image](https://user-images.githubusercontent.com/12745474/212639335-1361383c-a081-4af4-9ec0-a44452034dc8.png)